src/Voter/IsDueVoter.php

```diff
@@ -4,22 +4,23 @@
 
 namespace Synolia\SyliusSchedulerCommandPlugin\Voter;
 
+use Symfony\Component\DependencyInjection\Attribute\TaggedIterator;
+use Synolia\SyliusSchedulerCommandPlugin\Checker\IsDueCheckerInterface;
 use Synolia\SyliusSchedulerCommandPlugin\Components\Exceptions\Checker\IsNotDueException;
 use Synolia\SyliusSchedulerCommandPlugin\Entity\CommandInterface;
 
 class IsDueVoter implements IsDueVoterInterface
 {
-    /** @var array<\Synolia\SyliusSchedulerCommandPlugin\Checker\IsDueCheckerInterface> */
-    private array $isDueCheckers;
-
-    public function __construct(\Traversable $checkers)
-    {
-        $this->isDueCheckers = iterator_to_array($checkers);
+    public function __construct(
+        #[TaggedIterator(IsDueCheckerInterface::class)]
```

Comment:
deprecated since 7.1 will be removed in 8.x
https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/DependencyInjection/Attribute/TaggedIterator.php#L17